public/govwales-html-semantics.html

```diff
@@ -711,26 +711,24 @@ <h2>Site search</h2>
                     <div class="gw-col-xs-12">
                         <div class="gw-site-utilities__share-this-page">
                             <div id="gw-share-this-page" class="gw-share-this-page">
-                                <div class="gw-share-this-page__container">
-                                    <details id="gw-share-this-page-link">
-                                        <summary aria-controls="gw-share-this-page-link" aria-expanded="false">Share this page</summary>
-                                        <ul>
-                                            <li>
-                                                <a class="gw-social-media-link gw-social-media-link--x gw-social-media-link--sm" href="https://x.com/intent/post?url=%URL%"><span class="gw-visually-hidden">Share this page via</span> X</a>
-                                            </li>
-                                            <li>
-                                                <a class="gw-social-media-link gw-social-media-link--facebook gw-social-media-link--sm" href="https://www.facebook.com/sharer/sharer.php?u=%URL%">
-                                                    <span class="gw-visually-hidden">Share this page via</span> Facebook
-                                                </a>
-                                            </li>
-                                            <li>
-                                                <a class="gw-social-media-link gw-social-media-link--email gw-social-media-link--sm" href="mailto:?body=%URL%&amp;subject=Shared%20from%20%URL%">
-                                                    <span class="gw-visually-hidden">Share this page via</span> Email
-                                                </a>
-                                            </li>
-                                        </ul>
-                                    </details>
-                                </div>
+                                <details id="gw-share-this-page-link">
```

Comment:
Minor: Could we adjust this ID to "gw-share-this-page-links", same for the aria-controls on line 715.